.github/workflows/generate-msi.yml

```diff
@@ -19,4 +19,16 @@ jobs:
         run: echo "C:\Program Files (x86)\WiX Toolset v3.11\bin" >> $GITHUB_PATH
         shell: bash
       - name: Generate Pulumi MSI
-        run: dotnet run --project ./src -- generate msi
\ No newline at end of file
+        run: dotnet run --project ./src -- generate msi
+      - name: Setup dotnet SDK v5.0
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: '5.0.x'
+      - name: Push WinGet Manifest
```

Comment:
What will this do - does it publish something? This action runs on  push:

```
on: [push]
``` 

So presumably this will be executed on every push not just releases. Is that what we want? AFAIK winget.yml was only executed on releases.

Reply:
The code runs on every push but it checks whether it needs to deploy or not by itself:
 - It checks releases in pulumi/pulumi and compares them with releases in pulumi/winget
 - If there is a missing release here, it publishes it, writes a winget manifest file and opens a PR in winget packages repo, 
 - Otherwise, if the releases are up-to-date, it skips the process
 - Deleting the latest release here and renunning the GHA restarts the process

> probably only changes to master should trigger this

Reply:
Ah ok so this is like an idempotent sync! That should work then I think.